Hermes/hermes.py

```diff
@@ -100,6 +111,19 @@ def poll():
   # The poller can move forward without waiting for a response here.
   helper.urlfetch_async(request)
 
+def cache_node_stats():
+  try:
+    STATS['node'] = helper.update_node_cache()
+  except (AppControllerException, InfrastructureManagerException) as e:
```

Comment:
Pylint complains about single-letter variable names. I'm curious about the reason for this wrapper function instead of just catching the exceptions where they can be raised.